girder/utility/server.py

```diff
@@ -18,16 +18,34 @@
 ###############################################################################
 
 import cherrypy
+import mako
 import mimetypes
 import os
 import six
 
 import girder.events
 from girder import constants, logprint
+from girder.api.describe import API_VERSION
 from girder.utility import plugin_utilities, model_importer
 from girder.utility import config
 from . import webroot
 
+# TODO make a pretty error page someday.
+_errorTemplate = """
+<h1>${status}</h1>
+<p>${message}</p>
+<p><i>Girder server version ${version}</i></p>
```

Comment:
I'm not convinced that hiding the version provides any meaningful security. Even if we remove the version output everywhere, plenty of other public-facing aspects of a girder server can be used to gain information about what version it's running; front-end text, swagger specs, new endpoints that were added, the content of static files being served... Given that the source code is open, the task of figuring out the server version is pretty simple, and hence I don't think we can ever really consider Girder server version to be secure information. (Versions of other tools in our underlying stack can and should be hidden, though.)

The argument about search engines indexing version info actually makes *our* job easier more than anything, by making it easier for us to discover outdated versions deployed in the wild.

If this is a sticking point, let's discuss it offline and come to a consensus.